perf/jmh/build.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<project name="JMH" default="build" basedir=".">
+	<taskdef resource="net/sf/antcontrib/antlib.xml" />
+	<description>JMH</description>
+
+	<!-- set global properties for this build -->
+	<property name="DEST" value="${BUILD_ROOT}/perf/jmh" />
+	<property name="SRC" location="." />
+
+	<target name="init">
+		<mkdir dir="${DEST}" />
+	</target>
+
+	<target name="getJMH" depends="init" description="Download JMH">
+		<echo message="Cloning JMH repository"/>
+		<exec executable="git" failonerror="true" dir="${SRC}">
+			<arg line="clone https://github.com/openjdk/jmh-jdk-microbenchmarks"/>

```

Comment:
@karianna, Monica should be able to comment on this as well since I reached out to her. In my opinion, testing the write-barrier on both Intel and Arm architectures should be fine. The main intent of the testing is to compare the results between the two architectures, and as long as the testing is executed around the same time, it shouldn't create any problems for current requirements.